doc/contents.rst

```diff
@@ -7,6 +7,7 @@ Sphinx documentation contents
    :maxdepth: 2
 
    usage/index
+   tutorial/index
```

Comment:
If my understanding correct, this tutorial will replace the "Getting Started" page, right?
https://www.sphinx-doc.org/en/master/usage/quickstart.html

If so, this tree should be placed at the top of the document, I think.

Note: I understand this is still not matured. So I can agree to put this here until enough mature.

Reply:
Good point. Initially, I was not thinking about replacing the `quickstart.rst`, but it's true that, when it matures, it will be a better starting point.

I agree with moving it at the top, perhaps in a later pull request, when this has a bit more content?

Reply:
Okay, let's replace it with this after matured.